README.md

```diff
@@ -3,6 +3,12 @@
 This repository contains some infrastructure for the [Wikibase Registry][],
 which runs as a docker-compose setup in the `wbregistry-01` instance of the `wikibase-registry` Wikimedia Cloud VPS project.
 
+## Configuration
```

Comment:
Hm, I’m not a fan of having this section so early in the README, to be honest. As I understand it, this whole repository is (for now) only meant for exactly one system, which has already been set up – I’m worried that someone following this README will follow these instructions and overwrite the real configuration files that already exist :grimacing: 

How about “Initial configuration”? And maybe add a short sentence like “To initially configure a new system:”, similar to “To update the system:” below.

Reply:
Initial sounds good :+1: